Mathlib/FieldTheory/RatFunc.lean

```diff
@@ -814,9 +815,11 @@ instance (R : Type*) [CommSemiring R] [Algebra R K[X]] : Algebra R (RatFunc K) w
         ofFractionRing_mul, IsLocalization.mul_mk'_eq_mk'_of_mul, Algebra.smul_def]
   commutes' c x := mul_comm _ _
 
+instance : Coe (Polynomial K) (RatFunc K) := ⟨fun P ↦ algebraMap _ _ P⟩
+
 variable {K}
 
-theorem mk_one (x : K[X]) : RatFunc.mk x 1 = algebraMap _ _ x :=
+theorem mk_one (x : K[X]) : RatFunc.mk x 1 = algebraMap _ _ x := by
```

Comment:
Why adding the `by`?

Reply:
Old tentive thats stayed there, thanks for the catch.